.github/ISSUE_TEMPLATE/issue-default.yml

```diff
@@ -1,32 +1,32 @@
-name: Standard Issue Tempalte
-description: Create an Issue
-title: "[Issue]:"
-labels: ["research"]
-assignees:
-  - kitsushadow
+name: Issue
+description: Provide a title for the issue you are describing 
+title: "Please provide a clear title"
+labels: ["review"]
+# assignees:
+#   - kitsushadow
 body:
   - type: markdown
     attributes:
       value: |
-        Please fill this out best you can
+        Describe an issue, idea, task, bug, or story.
   - type: textarea
     id: why
     attributes:
       label: Why or Acceptance Criteria
-      description: What is the reason we determined we need an issue.
+      description: What is the reason it was determined this is an issue.
       placeholder: This issue is a product of...
-      value: "While working on X or researching Y it was discovered that Z was an issue that need captured."
+      value: "While working on X or researching Y it was discovered that Z was an issue that needed captured."
     validations:
       required: true
   - type: dropdown
     id: type
     attributes:
       label: Issue Type
-      description: What type of issue is this
+      description: What kind of role would take action on your issue?
       options:
         - research (Default)
-        - dev
-        - product
+        - development
+        - product team
```

Comment:
I could see this being useful. It is a bummer that dropdowns don't seem to integrate with things like labels just yet (https://github.com/orgs/community/discussions/4266 😢)

I might even suggest moving it closer to the top of the issue template so that it appears at a glance, setting the context for how we read the rest.